x-pack/legacy/plugins/siem/public/components/open_timeline/export_timeline/export_timeline.tsx

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import uuid from 'uuid';
+import { OpenTimelineResult } from '../types';
+import { GenericDownloader } from '../../generic_downloader';
+import * as i18n from '../translations';
+import { ActionListIcon, TimelineCustomAction } from '../delete_timeline_modal';
+import { useExportTimeline } from '.';
+import { useStateToaster } from '../../toasters';
+
+const ExportTimeline: React.FC<{
+  selectedTimelines: OpenTimelineResult[] | undefined;
+  onDownloadComplete?: () => void;
+}> = ({ selectedTimelines, onDownloadComplete }) => {
+  const { enableDownloader, setEnableDownloader, exportedIds, getExportedData } = useExportTimeline(
+    selectedTimelines
+  );
+  const [, dispatchToaster] = useStateToaster();
+  return (
+    <>
+      {selectedTimelines != null && exportedIds != null && enableDownloader && (
```

Comment:
I am wondering why `selectedTimelines` is part of the conditions?

Reply:
I think it works the same as the flag `enableDownloader`, agree to remove it and let the flag does its job.